yb-voyager/cmd/importData.go

```diff
@@ -101,6 +101,11 @@ var importDataCmd = &cobra.Command{
 		if err != nil {
 			utils.ErrExit("Error validating import flags: %s", err.Error())
 		}
+
+		err = validateImportDataFlags()
```

Comment:
Couple of things: 
- This is also something we'd want to do for import-data-file. We;ll have to add this validation there as well, right? 
- ImportDataCmd.PreRun is also called by `import-data-to-source-replica` and `import-data-to-source` ? Can we check if it's going to be a similar clash with those commands as well?

Reply:
> This is also something we'd want to do for import-data-file

Yeah correct, i assumed reusing import data command might be already taking care of it.
But thats not the case.

On digging further, i noticed that `validateImportFlags()` which has some common import data/file checks is not even called for `import data file`


Validations are missing for import data file cmd

Reply:
also some tests seem to be failing with a panic, pls check